packages/workspace/src/browser/workspace-preferences.ts

```diff
@@ -32,9 +32,9 @@ export const workspacePreferenceSchema: PreferenceSchema = {
             default: false
         },
         'workspace.supportMultiRootWorkspace': {
-            description: 'Enable the multi-root workspace support to test this feature internally',
+            description: 'Whether multi-root workspace support is enabled',
```

Comment:
[minor] maybe something more consistent with other preferences would be something like:

```json
'Controls whether multi-root workspace support is enabled.'
```

or

```json
'Enable multi-root workspace support.'
```